tutorial/ap_mfxp22820.yaml

```diff
@@ -0,0 +1,63 @@
+setup:
+  root_dir: '/cds/data/psdm/mfx/mfxp22820/scratch/apeck'
+  exp: 'mfxp22820'
+  run: 57
+  det_type: 'Rayonix'
+
+fetch_mask:
+  dataset: '/entry_1/data_1/mask'
+
+fetch_geom:
+
+build_mask:
+  thresholds: -10 5000
+  n_images: 20
+  n_edge: 1
+  combine: True
+
+run_analysis:
+  max_events: -1
+
+opt_geom:
+  n_iterations: 5
+  n_peaks: 3
+  threshold: 1000000
+  distance: 200
+
+find_peaks:
+  tag: ''
+  psana_mask: False
+  min_peaks: 10
+  max_peaks: 2048
+  npix_min: 2
+  npix_max: 30
+  amax_thr: 40.
+  atot_thr: 180.
+  son_min: 10.0
+  peak_rank: 3
+  r0: 3.0
+  dr: 2.0
+  nsigm: 10.0
+
+index:
+  tag: 'sample3'
+  int_radius: '3,4,5'
+  methods: 'mosflm'
+  cell: '/cds/data/psdm/mfx/mfxp22820/scratch/btx/sample.cell'
+  tolerance: '5,5,5,1.5'
+  no_revalidate: True
+  multi: True
+  profile: True
+
+stream_analysis:
+  tag: 'sample3'
+
+merge:
+  tag: 'sample3'
```

Comment:
I'm starting to think we could define an overall `setup.tag` and maybe have the option to overwrite it with `task.tag`?

Reply:
To be handled in a separate issue.